tools/ci-scripts/linux/build_cs.zsh

```diff
@@ -2,6 +2,8 @@
 
 set -eux
 
+nuget update -self
+nuget install NUnit.ConsoleRunner -outputdirectory /root -version 3.8.0 -NonInteractive
```

Comment:
If the `-ExcludeVersion` flag is passed to `install`, the output directory won't contain the version info, minimizing the amount of churn when we change versions.